staking/contracts/contracts.py

```diff
@@ -123,13 +138,13 @@ def calculate_rewards(addr: Expr) -> Expr:
     BareCallActions(
         # On create only, just approve
         no_op=OnCompleteAction.never(),
-        # Just be nice, we _must_ provide _something_ for clear state becuase it is its own
+        # Just be nice, we _must_ provide _something_ for clear state because it is its own
         # program and the router needs _something_ to build
         clear_state=OnCompleteAction.call_only(Approve()),
     ),
 )
 
-@router.method(no_op=CallConfig.ALL, opt_in=CallConfig.ALL)
+@router.method(no_op=CallConfig.CALL, opt_in=CallConfig.CALL)
```

Comment:
This is giving me a mypy error _likely_ due to a problem with PyTeal itself. In the context of smart contract development, I recommend maximum strictness including type checking. So type errors such as this would need to be explicitly ignored or dealt with in some other way.